repos/fdbt-site/src/interfaces/typeGuards.ts

```diff
@@ -163,9 +166,18 @@ export const isPeriodExpiry = (
     periodExpiryAttribute: PeriodExpiry | ErrorInfo[] | undefined,
 ): periodExpiryAttribute is PeriodExpiry => !!periodExpiryAttribute && 'productValidity' in periodExpiryAttribute;
 
+export const isDayOfTheWeek = (dayOfTheWeek: DayOfTheWeek | ErrorInfo[] | undefined): dayOfTheWeek is DayOfTheWeek =>
```

Comment:
I dont think we need this. The only place its used, the dayOfTheWeek input can only be either a DayOfTheWeek or undefined. This typeguard is only returning whether or not something is defined (!!dayOfTheWeek) - instead, we should just have a function which checks the input and makes sure it is one of the days of the week. 

For example

export const isADayOfTheWeek = (input: string): boolean => ['monday', 'tuesday', 'wednesday', 'thursday' , 'friday' , 'saturday' , 'sunday'].contains(input);

Reply:
The function should also live inside the API file, not in typeguards.